### PR TITLE
Create README.md (LEGAL-BERT Model card)

### PR DESCRIPTION
**Pull request by iliaschalkidis**
Model description for all LEGAL-BERT models, published as part of  "LEGAL-BERT: The Muppets straight out of Law School". Chalkidis et al., 2018, In Findings of EMNLP 2020

# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dimiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests), 
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? 

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: model_cards/nlpaueb/legal-bert-base-uncased/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# LEGAL-BERT: The Muppets straight out of Law School
```
Comment (julien-c) [inline model_cards/nlpaueb/legal-bert-base-uncased/README.md:1]:
```suggestion
---
language: en
tags:
- legal
---

# LEGAL-BERT: The Muppets straight out of Law School
```

**Comment by julien-c** [comment]
That's really cool, thanks for sharing @iliaschalkidis

**Comment by iliaschalkidis** [comment]
Thanks @julien-c for your nice comments and for building and improving such a great library. Is there any chance, that we could place all 5 LEGAL-BERT variants in a sub-folder, i.e., `/legal-bert`, inside the account folder `/nlpaueb`? Kind of OCD though 🤓

**Comment by julien-c** [comment]
I'm not sure what you mean :)

Do you want to e.g. rename `bert-base-uncased-contracts` to `legal-bert-base-uncased-contracts`? Or do you want `nlpaueb/legal-bert/bert-base-uncased-contracts`? We don't really want to do the latter IMO (have increased levels of nesting) because:
- I'm afraid it might get confusing for users of the models,
- some of the tooling we are currently building is expecting a org_name/model_name layout.

What do you think?

**Pull request merged**